### ISSUE TITLE
Help with getting auto logged out or keeping alerts on iOS enabled

### ISSUE THREAD

**Issue by fun4stuff** [twrecked/hass-aarlo#460]
I am able get aarlo installed without issue. 

The problem is that it auto logs me out of my Arlo account in the iOS app. 

So I created a new Arlo account and shared my cameras with this new account. 

I added the new Arlo account to configuration.yaml and secrets.yaml. I then restart home assistant server. 

For some reason, i still keep getting logged out out of my main/original account. Even if i log on  to a third Arlo account, i still do not get the motion alerts on my iOS phone. 

Can anyone help me troubleshoot this? I'm completely new to HA. Thanks!

**Comment by twrecked** [comment]
I think you found a bug in my new "save the token code". From your configuration directory try deleting ".aarlo/session.pickle". I bet it keeps trying to use your first configured login.

**Comment by fun4stuff** [comment]
Cool, I'll give that a try. Thanks for the reply!

**Comment by GJKrupa** [comment]
I don't think this is fixed. I'm running off master and my Android app is still getting logged out after a couple of seconds.
